docs/concepts/dotnet-standard-library.md

```diff
@@ -0,0 +1,130 @@
+
+The .NET Standard Library is not specific to any one .NET runtime, nor does it match the versioning scheme of any of those runtimes.
+
+APIs added to any of the runtimes (such as, .NET Framework, .NET Core and Mono) can be considered as candidates to add to the specification. New [versions of the .NET Standard Library](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md#list-of-net-corefx-apis-and-their-associated-net-platform-standard-version) are created for one of two reasons:
+
+- APIs are added to a .NET runtime (for example, .NET Core) that are considered fundamental. They are added to the .NET Standard Library, requiring other .NET runtimes to implement these new APIs in order to satisfy the new standard version.
+- APIs are implemented across a set of .NET runtimes (for example, .NET Framework, .NET Core and Mono) at the same time, making the APIs candidates for the standard.
+
+PCL Compatibility
+-----------------
+
+The .NET Standard Library is compatible with a subset of PCL profiles. .NET Standard Library 1.0, 1.1 and 1.2 each overlap with a set of PCL profiles. This overlap was created for two reasons:
+
+- Enable NSLs to reference PCLs.
+- Enable PCLs to be packaged as NSLs.
+
+PCL compatibility is provided by the [Microsoft.NETCore.Portable.Compatibility](https://www.nuget.org/packages/Microsoft.NETCore.Portable.Compatibility) NuGet package. This dependency is required when referencing NuGet packages that contain PCLs.
+
+You can see the set of PCL profiles that are compatible with the .NET Standard: 
+
+| Profile | .NET Platform Standard version |
+| ---------| --------------- |
+| Profile7  .NET Portable Subset (.NET Framework 4.5, Windows 8) | 1.1 |
+| Profile31 .NET Portable Subset (Windows 8.1, Windows Phone Silverlight 8.1)| 1.0 |
+| Profile32 .NET Portable Subset (Windows 8.1, Windows Phone 8.1) | 1.2 |
+| Profile44 .NET Portable Subset (.NET Framework 4.5.1, Windows 8.1) | 1.2 |
+| Profile49 .NET Portable Subset (.NET Framework 4.5, Windows Phone Silverlight 8) | 1.0 |
+| Profile78 .NET Portable Subset (.NET Framework 4.5, Windows 8, Windows Phone Silverlight 8) | 1.0 |
+| Profile84 .NET Portable Subset (Windows Phone 8.1, Windows Phone Silverlight 8.1) | 1.0 |
+| Profile111 .NET Portable Subset (.NET Framework 4.5, Windows 8, Windows Phone 8.1) | 1.1 |
+| Profile151 .NET Portable Subset (.NET Framework 4.5.1, Windows 8.1, Windows Phone 8.1) | 1.2 |
+| Profile157 .NET Portable Subset (Windows 8.1, Windows Phone 8.1, Windows Phone Silverlight 8.1) | 1.0 |
+| Profile259 .NET Portable Subset (.NET Framework 4.5, Windows 8, Windows Phone 8.1, Windows Phone Silverlight 8) | 1.0 |
+
+.NET Platforms Support
+======================
```

Comment:
I thought of this. Here's my quandry:

- PCL compability is part of the .NET Standard Library spec.
- .NET platforms support is not part of the spec, but is extrinsic.

That's what is influencing the order, currently.

Thoughts?